Sources/SWBUniversalPlatform/Plugin.swift

```diff
@@ -81,6 +81,6 @@ struct UniversalPlatformTaskProducerExtension: TaskProducerExtension {
 
 struct UniversalPlatformTaskActionExtension: TaskActionExtension {
     var taskActionImplementations: [SWBUtil.SerializableTypeCode : any SWBUtil.PolymorphicSerializable.Type] {
-        [41: TestEntryPointGenerationTaskAction.self]
+        [44: TestEntryPointGenerationTaskAction.self]
```

Comment:
@owenv The value of 41 was previously one higher than the largest type code in `BuiltinTaskActionsExtension`. I've increased it by 3 to account for the 1 new task action added here plus the 2 others that you just added. This seems fragile, though. If the intent is for it to have a unique code here, maybe we should put a placeholder in `BuiltinTaskActionsExtension` to reserve a number for this? I don't really have any context on this, though.